src/styles/globals.css

```diff
@@ -7,6 +7,10 @@
     /** reverts back to custom font after loading system font on slower connections  */
     font-display: swap;
     /** TODO better background color */
-    @apply bg-black;
+
+    /* @apply bg-black; */
```

Comment:
```suggestion
```

Reply:
Remove the extra space too.

![image](https://user-images.githubusercontent.com/89210438/211165427-f3290446-146c-49c2-bd60-b7e1f265c145.png)